src/main/java/io/vertx/core/net/NetServerOptions.java

```diff
@@ -54,11 +54,17 @@
    */
   public static final boolean DEFAULT_SNI = false;
 
+  /**
+   * Default value of whether the server supports HA Proxy protocol = false
+   */
+  public static final boolean DEFAULT_PROXY_SUPPORT = false;
+
   private int port;
   private String host;
   private int acceptBacklog;
   private ClientAuth clientAuth;
   private boolean sni;
+  private boolean proxySupport;
```

Comment:
rename it `proxyProtocolSupport` instead, proxy is confusing given that the client supports a `proxyOptions`